syft/pkg/cataloger/apkdb/package.go

```diff
@@ -27,7 +29,7 @@ func newPackage(d pkg.ApkMetadata, release *linux.Release, locations ...source.L
 }
 
 // packageURL returns the PURL for the specific Alpine package (see https://github.com/package-url/purl-spec)
-func packageURL(m pkg.ApkMetadata, distro *linux.Release) string {
+func packageURL(m parsedData, distro *linux.Release) string {
```

Comment:
this could have stayed as pkg.ApkMetadata (there isn't anything on parsedData that is needed here)